src/main/webapp/createUsers.js

```diff
@@ -0,0 +1,65 @@
+/*$(document).on('click', document.getElementById('update_users'), function () {
+  $.get(window.location.origin + '/users', usersFetch());
+});*/
+
+/*$("#update_users").click(function (e) {
+  e.preventDefault();
+  $.ajax({
+    url: window.location.origin + "/users",
+    type: "GET",
+  }).done(usersFetch());
+});*/
+
+$('#update_users').click(function (e) {
+  e.preventDefault();
+  $.ajax({
+    url: window.location.origin + '/users',
+    type: 'GET',
+  }).done(usersFetch());
+});
+
+async function usersFetch() {
+  //add a new user onto the dashboard based on how many are returned
+  const users = await (await fetch('/users')).json();
+  const userDashboard = document.getElementById('profile-grid');
+  console.log('here');
+  users.forEach((user) => {
+    console.log(user);
+    userDashboard.appendChild(createUserCard(user));
+    console.log('user checked');
+  });
+}
+
+//create a user template
+function createUserCard(user) {
+  //make an empty card element
+  const UserCard = document.createElement('div');
```

Comment:
[Local variable names are written in lowerCamelCase](https://google.github.io/styleguide/jsguide.html#naming-local-variable-names)

Reply:
Still needed